var/spack/repos/builtin/packages/netcdf-cxx4/package.py

```diff
@@ -43,7 +44,17 @@ def flag_handler(self, name, flags):
             flags.append(self.compiler.cc_pic_flag)
         elif name == 'cppflags':
             flags.append('-I' + self.spec['netcdf-c'].prefix.include)
-
+        elif name == 'ldflags':
```

Comment:
I think we don't need this because Spack does not install the `*.la` files for quite some time already.  Please, feel free to remove this logic from `netcdf-fortran` too.